src/Plugin/Field/FieldFormatter/CaptureUtilityMapFormatter.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\web_page_archive\Plugin\Field\FieldFormatter;
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Field\FieldItemInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\FormatterBase;
+
+/**
+ * Plugin implementation of the 'web_page_archive_capture_formatter' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "web_page_archive_capture_utility_map_formatter",
+ *   label = @Translation("Web page archive capture utility map formatter"),
+ *   field_types = {
+ *     "map"
+ *   }
+ * )
+ */
+class CaptureUtilityMapFormatter extends FormatterBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(FieldItemListInterface $items, $langcode) {
+    $elements = [];
+    foreach ($items as $delta => $item) {
+      $elements[$delta] = ['#markup' => $this->viewValue($item)];
+    }
+
+    return $elements;
+  }
+
+  /**
+   * Generate the output appropriate for one field item.
+   *
+   * @param \Drupal\Core\Field\FieldItemInterface $item
+   *   One field item.
+   *
+   * @return string
+   *   The textual output generated.
+   */
+  protected function viewValue(FieldItemInterface $item) {
+    return nl2br(Html::escape(implode("\n", array_values($item->toArray()))));
```

Comment:
Ah yeah, it's less to do with it being an array, and more about the `Html::escape()` method.  Take a look at _any_ of the Drupal core formatter `viewValue()` methods, none of them sanitize the output.  It's the same thinking as retrieving information out of the database, as you don't know what someone might want to do with it, you leave it to them to sanitize it themselves, else provide a setting/ switch to implement it as an option in the UI;
http://cgit.drupalcode.org/computed_field/tree/src/Plugin/Field/FieldFormatter/ComputedStringFormatter.php?h=8.x-2.x
```
  public function settingsForm(array $form, FormStateInterface $form_state) {
    return [
      'sanitized' => [
        '#type' => 'checkbox',
        '#title' => t('Sanitized'),
        '#default_value' => $this->getSetting('sanitized'),
      ]
    ] + parent::settingsForm($form, $form_state);
  }

  protected function prepareValue($value) {

    if ($this->getSetting('sanitized')) {
      return nl2br(Html::escape($value));
    } else {
      return nl2br($value);
    }
  }
```